### PR TITLE
Fix version check for case of "1.8-dev"

### PR DESCRIPTION
**Pull request by nickbp**
Stays pretty strict for now, supporting only '-dev', and only on the second element.

Moves version element extraction upstream in case other utils need it someday.

### CODE REVIEW

File: src/main/java/org/apache/mesos/dcos/DcosVersion.java [hunk 1]
```diff
@@ -53,4 +54,26 @@ public String getDcosImageCommit() {
     public String getVersion() {
         return version;
     }
+
+    public int getVersionFirstElement() throws NumberFormatException {
```
Comment (gabrielhartmann) [inline src/main/java/org/apache/mesos/dcos/DcosVersion.java]: Can we encapsulate Version in a Class?  So we get something like:
```
public Version getVersion();
Version version = getVersion();
int firstElement = version.getFirstElement();
int secondElement = version.getSecondElement();
...
```

Comment (nickbp) [inline src/main/java/org/apache/mesos/dcos/DcosVersion.java]: Done

File: src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java [hunk 2]
```diff
@@ -12,7 +12,6 @@
  */
 public class CapabilitiesTest {
```
Comment (gabrielhartmann) [inline src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java]: @nickbp:  Thanks for all the tests.  Can you add one more that exercises the throwing of the NumberFormatException please?  So something that has version returned that looks like `1` for example and maybe something with `0.` as another example.

Comment (nickbp) [inline src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java]: Done

**Pull request merged**